Formula/c/cargo-release.rb

```diff
@@ -24,7 +22,6 @@ class CargoRelease < Formula
   depends_on "libgit2"
 
   def install
-    ENV["LIBGIT2_NO_VENDOR"] = "1"
```

Comment:
we might still want to keep this line

Reply:
see https://github.com/rust-lang/git2-rs/blob/master/libgit2-sys/build.rs#L30-L49

Reply:
That was a mistake on my part. Will revert.